hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -242,14 +242,14 @@ class HassioAddonInfo extends LitElement {
               ? html`
                   Current version: ${this.addon.version}
                   <div class="changelog" @click=${this._openChangelog}>
```

Comment:
Why do we have this div anyway?!

Reply:
🤷 `<button class='link'>` would be better.